### ISSUE TITLE
Please tag your releases in github

### ISSUE THREAD

**Issue by niol** [cyface/django-termsandconditions#25]
pip has 1.1.14 which is not tagged in github. This makes it hard to follow what is part of the release. Or is development happenning elsewhere?

Thanks a lot.

**Comment by cyface** [comment]
Done, thanks!

**Comment by cyface** [comment]
Thanks, I updated the tag name!

**Comment by niol** [comment]
Thanks a lot, the tag without v is still there FYI.

**Comment by cyface** [comment]
Fixed, thanks!
